src/algorithms/tracking/TrackerMeasurementFromHits.cc

```diff
@@ -60,15 +60,13 @@ TrackerMeasurementFromHits::produce(const edm4eic::TrackerHitCollection& trk_hit
     const auto* vol_ctx = m_converter->findContext(hit.getCellID());
     auto vol_id         = vol_ctx->identifier;
 
-    auto surfaceMap = m_acts_context->surfaceMap();
```

Comment:
Can we instead make the `surfaceMap` into a reference `& surfaceMap` here?

Reply:
Sure, done.

I haven't checked running with logging so that might still be unnecessarily inefficient, but less important.